### DISCUSSION TITLE
Blank "Loading..." screen after fresh install

### DISCUSSION THREAD

**Discussion by VicFleur** [directus/directus#4556] [Archive]
Fresh installation of Directus today in a Plesk environment. Followed the steps that got me a working Directus copy last week: 
- ```npx create-directus-project cms```, which then gets stuck at ```/lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found``` but can be solved by ```npm install``` and  ```npm rebuild argon2 --build-from-source```
- ```npx directus init``` which works well and correctly sets up the .env file
- ```npx directus start``` which correctly says the server is started at port 8055

The whole cms directory and its database where removed prior to this new installation because I wanted to start afresh. 

This is my nginx configuration, which again, was working fine last week:
```
location ~ / {
	proxy_redirect off;
	proxy_set_header Host $http_host;
	proxy_set_header X-Real-IP $remote_addr;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header X-Forwarded-Proto $scheme;
	proxy_read_timeout 1m;
	proxy_connect_timeout 1m;
	proxy_pass http://localhost:8055;
}
```

Now, when going to ```/``` I get redirected to ```/admin``` and get a blank screen. Title says "Loading..." and I can see Directus' favicon. 

Browser console logs the following errors:
```
Did not parse stylesheet at ' (redacted url) /admin/css/chunk-vendors.d810bbd2.css' because non CSS MIME types are not allowed in strict mode.
SyntaxError: Unexpected token '<' chunk-vendors.60bda00b.js:1
```

The content of "chunk-vendors.60bda00b.js:1" is the same "Loading..." html page I'm seeing

I can't seem to be able to do anything else. Reinstalling again didn't help. Can confirm I'm using node 12.21 in my terminal for everything mentioned above. Changing browser didn't help.

Any ideas?

**Environment:**
- CentOS7
- Plesk Obsidian 18.0.33
- npm v6.14.11
- node v12.21.0
- PostgreSQL 9.2.24-4.el7_8

**Comment by rijkvanzanten** [comment]
Hmm, based on `SyntaxError: Unexpected token '<' chunk-vendors.60bda00b.js:1`, I would assume that your webserver (in this case nginx) is forwarding every request to `/`, which in turn serves `/admin` which is HTML (eg `<`). 

Could you try using the following?:

```
    location / {
        proxy_pass http://localhost:8055;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection 'upgrade';
        proxy_set_header Host $host;
        proxy_cache_bypass $http_upgrade;
    }
```

(i'm no nginx expert, but i've had success with the above config in the past)
